source/components/element/HxElement.less

```diff
@@ -0,0 +1,5 @@
+@import (reference) "vars";
+
+* {
+  box-sizing: border-box;
```

Comment:
This is what's needed to correct style bugs. Especially with redistributed elements containing grid classes.